src/Microsoft.Data.SqlClient/tests/ManualTests/DataCommon/ConnectionTestParametersData.cs

```diff
@@ -15,6 +15,8 @@ namespace Microsoft.Data.SqlClient.ManualTesting.Tests.DataCommon
 {
     public class ConnectionTestParametersData
     {
+        private const int CASES = 30;
+        private string _empty = string.Empty;
```

Comment:
If you do not specify HNIC its default value is string.Empty no need to reassign it again.

Reply:
This statement does not make sense.  The ConnectionTestParametersData fields must all be initialized for each test case.